tests/contrib/ray/__init__.py

```diff
@@ -0,0 +1,19 @@
+from ray.util.tracing import tracing_helper
+from ray.util.tracing.tracing_helper import _is_tracing_enabled
+
+
+def setup_test_tracing():
+    from ddtrace import config
+
+    config.ray.trace_core_api = True
+
+    from ddtrace import patch
+    import ddtrace.auto  # noqa:F401
```

Comment:
is this for testing a specific behavior or how you expect users to use this?

Reply:
addressed in [051f9e5](https://github.com/DataDog/dd-trace-py/pull/14742/commits/051f9e5b1cb3ac24e3b1e4172a198151b99a9674)